src/main/java/lotto/domain/LottoStore.java

```diff
@@ -10,15 +10,22 @@
     private LottoStore() {
     }
 
-    public static Lottos purchase(Money money) {
-        List<Lotto> lottos = createLottos(money.divide());
+    public static Lottos purchase(final Money money, final int manualPurchaseCount, final List<List<Integer>> manualLottoNumber) {
+        List<Lotto> lottos = new ArrayList<>();
+        if (isExistManualLotto(manualPurchaseCount)) {
+            manualLottoCountCheck(manualPurchaseCount, manualLottoNumber);
+            manualLottoMoneyExcessCheck(manualPurchaseCount, money);
+            lottos.addAll(createManualLottos(manualLottoNumber));
+        }
+        int autoIssuanceCount = money.divide() - manualPurchaseCount;
```

Comment:
LOTTO_PRICE 상수의 위치는 Money보다는 LottoStore가 더욱 적합해 보이고 LOTTO_PRICE 타입도 Money로 포장해보는건 어떨까요?

```suggestion
        int autoIssuanceCount = money.divide(LOTTO_PRICE) - manualPurchaseCount;
```